src/interactive-window/interactiveWindow.ts

```diff
@@ -200,7 +200,7 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
             }
         }, this.internalDisposables);
 
-        if (window.activeNotebookEditor === this.notebookEditor) {
+        if (window.activeNotebookEditor?.notebook.uri.toString() === this.notebookUri.toString()) {
```

Comment:
No need to change, however you could compare the instances of the Notebook objects as well, as that will be the same.